### DISCUSSION TITLE
2 differents locations

### DISCUSSION THREAD

**Discussion by zeze1959** [Stellarium/stellarium#2270] [Q&A]
Hi,
I´m new here and I want know if it´s possible to have 2 differents location in Stellarium? One in North hemisphere and other in South hemisphere?

**Comment by gzotti** [comment]
Not at the same time. Press F1 for an introduction, read the first part of the User Guide for a thorough start, or press F6  or the wind rose icon to select your location.

**Comment by zeze1959** [comment]
Hello gzotti,
Even if I create another shortcut, one for each loacation, it´s possible to work?

**Reply by gzotti** [reply]
If you use the command-line options in the User Guide, Ch.6, you can add or modify shortcuts with them, yes. 

Or you use the --config-file argument with two different settings. 

You can also have two entirely different installations and use the command-line argument --user-dir to point to that directory with the config.ini etc. you want to use. 

You can even run both versions at the same time and synchronize them with the RemoteSync plugin (Be sure to not synchronize the locations, obviously!)
